collect_app/src/androidTest/java/org/odk/collect/android/database/helpers/InstancesDatabaseHelperTest.java

```diff
@@ -43,14 +47,13 @@ public void restoreRealDb() {
         FileUtils.copyFile(new File(DATABASE_PATH + TEMPORARY_EXTENSION), new File(DATABASE_PATH));
     }
 
-    @Parameterized.Parameters(name = "{0}")
+    @Parameterized.Parameters(name = "{1}")
     public static Iterable<Object[]> data() {
         return Arrays.asList(new Object[][] {
-                {"Downgrading from version with extra column drops that column", "instances_v7000_added_fakeColumn.db"},
-                {"Downgrading from version with missing column adds that column", "instances_v7000_removed_jrVersion.db"},
+                {DOWNGRADE, "Downgrading from 5+ to version 5 should updated database paths", "instances_v6.db"},
```

Comment:
How about making this a separate test case? The other assertions aren't relevant to it and it's exercising particular behavior that will need to be removed in a few versions when downgrade doesn't touch the paths anymore.

Reply:
Done.